src/components/Tabs/Tab/Tab.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+import { isSelectKey } from '../utils';
+import style from './Tab.st.css';
+
+export interface TabItem {
+  /** Title of the tab */
+  title?: string;
+}
+
+interface TabProps {
+  item: TabItem;
+  index: number;
+  isActive: boolean;
+  onClick(index: number): void;
+  className: string;
+  dataHook: string;
+}
+
+export const Tab = React.forwardRef<HTMLDivElement, TabProps>((props, ref) => {
+  const { index, item, onClick, isActive, dataHook } = props;
+  const title = item.title;
+  const onSelectTab = () => {
+    onClick(index);
+  };
+
+  const onKeyDown = (e: React.KeyboardEvent<HTMLDivElement>) => {
+    const keyCode = e.keyCode;
+
+    if (isSelectKey(keyCode)) {
+      onSelectTab();
+      return false;
+    }
+  };
+
+  return (
+    <div
+      {...style('root', {}, props)}
+      ref={ref}
+      data-index={index}
+      data-active={isActive}
+      data-hook={dataHook}
+      key={`${title}-${index}`}
+      onClick={onSelectTab}
+      onKeyDown={onKeyDown}
+      tabIndex={0}
```

Comment:
It make sense to put same tabIndex for the inner tabs (probably the same as the parent or parent + 1) but in storyBook tabs doesn't seem to work - needs to be tested and implemented.

Reply:
I'm not sure the parent should be focusable. 
Nevertheless, `tabindex={0}` will work with the natural structure of the component